sample_apps/simple_app_python/example_app.py

```diff
@@ -163,12 +164,6 @@ def do_main(args) -> bool:
             print(fnl(), 'certify_me() failed\n')
             sys.exit(1)
 
-        # Server will persist required files upon certification.
-        if appln_data_dir.endswith(SERVER_APP_DATA):
-            whoami = 'server'
-            # Persist cert-and-key files in the provisioning dir.
-            write_cert_pvt_key_to_files(cctm, whoami, PROVISIONING_DIR, print_all)
```

Comment:
Reworked this logic, so that we now can show that client & server can be re-started. And that `warm_restart()` correctly re-installs trust data from policy-store.

Hence, we no longer write-out certificate data upon initial `get-certified` certification. This will be re-written-out each time server & / or client-app is restarted.